lib/crypto/src/crypto.erl

```diff
@@ -605,30 +653,36 @@
 %%   error:{badarg,Reason::term()}
 %%   error:{notsup,Reason::term()}
 %%   error:{error,Reason::term()}
--type descriptive_error() :: any() .
-
+-type descriptive_error() :: any().
 
 %%--------------------------------------------------------------------
 %%
 %% Make the new descriptive_error() look like the old run_time_error()
 %%
 -define(COMPAT(CALL),
-        try begin CALL end
-        catch
-            error:{error, {_File,_Line}, _Reason} ->
-                error(badarg);
-            error:{E, {_File,_Line}, _Reason} when E==notsup ; E==badarg ->
-                error(E)
-        end).
+    try
+        begin
+            CALL
+        end
+    catch
+        error:{error, {_File, _Line}, _Reason} ->
```

Comment:
One of my favorite opinions: Colde should be readable.  You read code much more often than you write it.  Therefore the reader should be helped:
 `{A, {B, C}, D} ` looks like arity four, while
`{A, {B,C},  D}`  shows more clearly that it is an arity three tuple with a tuple as second argument

Reply:
Yes, to formulate a rule telling this is nice or not nice is not easy...